backend/data/src/main/kotlin/io/tolgee/service/TelemetryService.kt

```diff
@@ -27,12 +30,17 @@ class TelemetryService(
     val data: TelemetryReportRequest = getTelemetryData()
     if (data.projectsCount == 0L) return
     if (data.usersCount == 0L) return
-    httpClient.requestForJson(
-      "${telemetryProperties.server}/v2/public/telemetry/report",
-      data,
-      HttpMethod.POST,
-      Unit::class.java,
-    )
+    try {
+      httpClient.requestForJson(
+        "${telemetryProperties.server}/v2/public/telemetry/report",
+        data,
+        HttpMethod.POST,
+        Unit::class.java,
+      )
+    } catch (e: Throwable) {
```

Comment:
``` suggestion
    } catch (e: Exception) {
```

Catch exception only. Catching errors as well is probably a bad idea.

Reply:
Noob question... but... Why is it bad idea? 🤔

Reply:
It is usually discouraged because it is generally impossible to recover from such errors programmatically anyway, and ignoring them (by catching them) only makes the issue worse. See: https://stackoverflow.com/a/2274130.